docs/deploy/agent-engine/test.md

```diff
@@ -0,0 +1,294 @@
+with the agent using REST calls or the Vertex AI SDK for Python. These instructions
+explain how test an ADK agent deployed in Agent Engine.
+
+## View deployed agent in Cloud Console
+
+To view your deployed agent in the Cloud Console:
+
+-   Navigate to the Agent Engine page in the Google Cloud Console:
+    [https://console.cloud.google.com/vertex-ai/agents/agent-engines](https://console.cloud.google.com/vertex-ai/agents/agent-engines)
+
+This page lists all deployed agents in your currently selected Google Cloud
+project. If you do not see your agent listed, make sure you have your
+target project selected in Google Cloud Console. For more information on
+selecting an exising Google Cloud project, see
+[Creating and managing projects](https://cloud.google.com/resource-manager/docs/creating-managing-projects#identifying_projects).
+
+## Find Google Cloud project information
+
+You need the address and resource identification for your project (`PROJECT_ID`,
+`LOCATION`, `RESOURCE_ID`) to be able to test your deployment. You can use Cloud
+Console or the `gcloud` command line tool to find this information.
+
+!!! note "Vertex AI express mode API key"
+    If you are using Vertex AI express mode, you can skip this step and use your API key.
+
+To find your project information with Google Cloud Console:
+
+1.  In the Google Cloud Console, navigate to the Agent Engine page:
+    [https://console.cloud.google.com/vertex-ai/agents/agent-engines](https://console.cloud.google.com/vertex-ai/agents/agent-engines)
+
+1.  At the top of the page, select **API URLs**, and then copy the **Query
+    URL** string for your deployed agent, which should be in this format:
+
+        https://$(LOCATION_ID)-aiplatform.googleapis.com/v1/projects/$(PROJECT_ID)/locations/$(LOCATION_ID)/reasoningEngines/$(RESOURCE_ID):query
+
+To find your project information with `gloud`:
+
+1.  In your development environment, make sure you are authenticated to
+    Google Cloud and run the following command to list your project:
+
+    ```shell
+    gcloud projects list
+    ```
+
+1.  Take the Project ID used for deployment and run this command to get
+    the additional details:
+
+    ```shell
+    gcloud asset search-all-resources \
+        --scope=projects/$(PROJECT_ID) \
+        --asset-types='aiplatform.googleapis.com/ReasoningEngine' \
+        --format="table(name,assetType,location,reasoning_engine_id)"
+    ```
+
+## Test using REST calls
+
+A simple way to interact with your deployed agent in Agent Engine is to use REST
+calls with the `curl` tool. This section describes the how to check your
+connection to the agent and also to test processing of a request by the deployed
+agent.
+
+### Check connection to agent
+
+You can check your connection to the running agent using the **Query URL**
+available in the Agent Engine section of the Cloud Console. This check does not
+execute the deployed agent, but returns information about the agent.
```

Comment:
This is inconsistent, if you click on "Copy query URL" from within an agent instance in the Cloud Console, you get a URL with `:query` at the end, which will query and execute the deployed agent. The user would need to remove `:query` from the end of that URL to match the form of the "get" request as shown in the command.